kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -109,10 +166,22 @@ this will then trigger the successful processing of the traveller and put anothe
 }
 ```
 
+* To examine ProcessedTravellers topic and verify it was auto sent
+
+```
+kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic processedtravellers --from-beginning
+```
+
 there are bunch of extension attributes that starts with `kogito` to provide some context of the execution and the event producer.
 
 To take the other path of the process put following message on `travellers` topic
 
+* Send Message to Topic
```

Comment:
`Send message that should be skipped to Topic`

Reply:
Done @radtriste